ci/openshift-ci/images/Dockerfile.buildroot

```diff
@@ -6,4 +6,9 @@
 #
 FROM registry.centos.org/centos:8
 
-RUN yum -y update && yum -y install git sudo wget
+RUN yum -y update && \
+    yum -y install \
+    git \
+    sudo \
+    wget && \
```

Comment:
Nit: We should really be using `curl`, but not strictly required by this PR (but you could add another commit).